Sources/joyfill-components-swift/Components/ComponentViewController/APIComponents/ComponentViewController.swift

```diff
@@ -109,8 +103,8 @@ extension ComponentsViewController: UITableViewDelegate, UITableViewDataSource {
                 
                 // MARK: MultiChoice Function Call From Package
                 let multipleChoice = MultipleChoice()
-                tableView.rowHeight = 200
-                multipleChoice.titleFontSize = 20
+                tableView.rowHeight = 190
+                multipleChoice.multiSelect = true
```

Comment:
Why are there pre-set values here in the renderer? Wouldn't things like this be in the components and then options for this field would be passed in.

Reply:
It was missed on my side. I've corrected with the api